frontend/src/locales/es.json

```
@@ -34,10 +35,11 @@
   "contribution": {
     "activity": "Actividad",
     "alert": {
+      "answerQuestion": "Please answer the question",
       "communityNoteList": "Aquí encontrarás todas las contribuciones enviadas y confirmadas de todos los miembros de esta comunidad.",
       "confirm": "confirmado",
+      "in_progress": "Es gibt eine Rückfrage der Moderatoren.",
```

```suggestion
      "in_progress": "There is a question from the moderators.",
```

May @Mogge could translate it?